src/components/admin/MainIssueController.jsx

```diff
@@ -318,12 +332,19 @@ export default class MainIssueController extends FalcorController {
       },
     };
     // Fill in the data
+    const publishedAt = moment(this.state.published_at).format('YYYY-MM-DD');
+    jsonGraphEnvelope.jsonGraph.issues.byNumber[issueNumber].published_at =
+      publishedAt;
```

Comment:
I'm pretty sure we usually just store it as a number... this could possibly work as well though